### PR TITLE
Adding new dApp

### PR DESCRIPTION
**Pull request by FedererKK**
##### Name (to be shown on DefiLlama):
Deribet

##### Twitter Link:
https://twitter.com/deribet_io

##### Website Link:
https://deribet.io/

##### Logo (High resolution, will be shown with rounded borders):
image

##### Current TVL:
Around 20k$

##### Chain:
Base

##### Short Description (to be shown on DefiLlama):
Memecoin fast paced prediction market and gaming

##### Token address and ticker if any:
Not yet

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Gaming

##### methodology (what is being counted as tvl, how is tvl being calculated):
Sum of dollar value of all memecoins

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/deribet: 
        
 
  
    
    
    TypeError: i.toLowerCase is not a function
    Truncated error stack:
    
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/tokenMapping.js:85:50
        at Array.forEach (<anonymous>)
        at getUniqueAddresses (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/tokenMapping.js:85:13)
        at sumTokens2 (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/unwrapLPs.js:713:12)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/unwrapLPs.js:830:25
        at getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:49:29)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:138:15
        at Array.map (<anonymous>)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:126:29
        at Array.map (<anonymous>)

**Comment by llamatester** [comment]
The adapter at projects/deribet exports TVL: 
        
 
 
    base                      0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/deribet exports TVL: 
        
 
 
    base                      24.26 k
    
    total                    24.26 k

File: projects/deribet/index.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+const { sumTokens2, nullAddress } = require("../helper/unwrapLPs");
+const NATIVE_ADDRESS = "NATIVE";
+
+const owners = [
+  "0x9eb71114b8cccd2b2c6097cf8c64e85c5d79d2a3",
```
Comment (g1nt0ki) [inline projects/deribet/index.js:5]: this is not a smart contract, we cant count the value here as tvl

**Review by g1nt0ki — CHANGES_REQUESTED** [review]
.

**Pull request closed without merging**